src/client/components/voyage/StartVoyage.js

```diff
@@ -0,0 +1,241 @@
+    const optimisation_type = {
+      optimisation_type: this.state.optimisationType[selectedIndex]
+        .optimisation_type
+    };
+    this.setState({ optimisation_type: optimisation_type });
+  };
+
+  render() {
+    const {
+      departureDate,
+      departureTime,
+      arrivalDate,
+      arrivalTime,
+      hire_rate,
+      forward_draft,
+      aft_draft,
+      lfso_cost,
+      hfo_cost
+    } = this.state;
+
+    const {
+      vessels,
+      ports,
+      optimisationType,
+      min_Draft,
+      max_Draft,
+      draft_StepSize,
+      draft_Unit,
+      min_fuelCost,
+      max_fuelCost,
+      fuelCost_stepSize,
+      fuelCost_Unit,
+      min_rate,
+      max_rate,
+      rate_stepSize,
+      rate_Unit
+    } = this.props;
+
+    let checkBoxLogic;
+    if (this.state.isChecked === true) {
+      checkBoxLogic = (
+        <InputField
+          label=" Hire Rate "
+          name="hire_rate"
+          min={min_rate}
+          max={max_rate}
+          step={rate_stepSize}
+          unit={rate_Unit}
+          value={hire_rate}
+          onDataInputChange={this.handleDataInputChange}
+        />
+      );
+    }
+    return (
+      <div>
+        <form>
+          <Title title=" Start Voyage " />
+          <br />
+          <Dropdown
+            label=" Choose vessel "
+            optionsMap={vessels}
+            handleSelection={this.handleVesselSelection}
+          />
+          <br />
+          <Checkbox
+            label=" Chartered vessel "
+            handleCheckbox={this.handleToggleCheckbox}
+          />
+
+          {checkBoxLogic}
+
+          <br />
+          <Dropdown
+            label=" Depart from "
+            optionsMap={ports}
+            handleSelection={this.handleDeparturePortSelection}
+          />
+          <br />
+          <DateTime
+            label=" Estimated Departure (ETD) "
+            name="ETD"
+            date={departureDate}
+            time={departureTime}
+            onDateInputChange={this.handleDateInputChange}
+            onTimeInputChange={this.handleTimeInputChange}
+          />
+          <br />
+          <Dropdown
+            label=" Destination "
+            optionsMap={ports}
+            handleSelection={this.handleArrivalPortSelection}
+          />
+          <br />
+          <DateTime
+            label=" Estimated Arrival (ETA) "
+            name="ETA"
+            date={arrivalDate}
+            time={arrivalTime}
+            onDateInputChange={this.handleDateInputChange}
+            onTimeInputChange={this.handleTimeInputChange}
+          />
+          <br />
+          <Dropdown
+            label=" Optimisation type "
+            optionsMap={optimisationType}
+            handleSelection={this.handleOptimisationTypeSelection}
+          />
+          <br />
+          <InputField
+            label=" Forward Draft "
+            name="forward_draft"
+            min={min_Draft}
+            max={max_Draft}
```

Comment:
Please follow same naming convention through out the file like max_Draft or max_draft.
make sure all having same style.

Reply:
Comment resolved, adopted same naming convention through out the file.